pkg/openevec/config.go

```diff
@@ -73,6 +82,7 @@ type AdamConfig struct {
 	CertsEVEIP  string `mapstructure:"eve-ip" cobraflag:"eve-ip"`
 	APIv1       bool   `mapstructure:"v1" cobrafalg:"force"`
 	Force       bool   `mapstructure:"force" cobraflag:"force"`
+	CA          string `mapstructure:"ca"`
```

Comment:
it is used anywhere?

Reply:
Needed for backward compatibility with escript tests, some of them use it.